src/clean.js

```diff
@@ -107,12 +102,9 @@ function clean(node, newObj) {
     newObj.name = newObj.name.replace(/^\\/, "");
   }
 
-  // TODO: remove after resolve https://github.com/glayzzle/php-parser/issues/181
-  if (node.kind === "unary" && ["+", "-"].includes(node.type)) {
-    return {
-      kind: "number",
-      value: `${newObj.type}${newObj.what.value}`
-    };
+  // @TODO: We need special node for `null` value to avoid ast compare problem
```

Comment:
Maybe link to an issue (and create it if it doesn't exist yet?)

Reply:
@czosel I'm not sure yet we really need a new node or `null` should be handle as identifier, i need investigate and then create issue if it is needed.

Reply:
Alright :+1: